snap/bin/stop-edgex.sh

```diff
@@ -95,12 +95,30 @@ if [ $SUPPORT_LOGGING = "y" ] ; then
     fi
 fi
 
+if [ $SECURITY ]; then
+    # stop kong - note this might fail, so we just force it to always succeed with "|| true"
```

Comment:
Why could this fail?  Shouldn't we use a SIGKILL if the polite stop fails?

Reply:
Well so firstly I don't know why it doesn't fail, but kong isn't the process name that makes up "kong" the kong process is really nginx, so we do use SIGINT to nginx, which respects SIGINT better.

What would happen sometimes is that nginx is still running, i.e. kong is still running, but for some reason the environment file that is located in the prefix location of `$SNAP_DATA/kong` would be invalid or deleted or something and so this kong command wouldn't effectively kill the nginx processes. Hence I also make the SIGINT to nginx below.